src/elements/Input/Input.tsx

```diff
@@ -317,7 +317,7 @@ export const Input = forwardRef<InputRef, InputProps>(
       }
     }, [fontFamily, space, color])
 
-    const inputVariants = getInputVariants(theme)
+    const inputVariants = useMemo(() => getInputVariants(theme), [])
```

Comment:
Maybe we could update the dependencies for the useMemo here?

```suggestion
    const inputVariants = useMemo(() => getInputVariants(theme), [theme])
```